assets/js/vue/Stats.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="row">
+        <div class="col-xl-12">
+            <repository-summary />
+        </div>
+        <div class="col-xl-12">
+            <dataset-over-time />
+        </div>
+        <div class="col-xl-12">
+            <dataset-size-ranges />
+        </div>
+    </div>
+</template>
+
+<script>
+    const axios = require('axios');
+    import DatasetOverTime from "./components/stats/DatasetOverTime";
+    import DatasetSizeRanges from "./components/stats/DatasetSizeRanges";
+    import RepositorySummary from "./components/stats/RepositorySummary";
+
+    export default {
+        name: "StatsApp",
+        components: { DatasetOverTime, DatasetSizeRanges, RepositorySummary },
+          methods: {
+            getThemeColor: function(color) {
```

Comment:
This method is never used in the parent component, Instead of this I think we could use Vue mixins
https://vuejs.org/v2/guide/mixins.html, which can have a method that gets the theme color

Reply:
I have to change that a bit anyway, because the color will not be the same "name" for each template.